src/common/plugin/pluginState.ts

```diff
@@ -99,7 +99,7 @@ export class PluginState extends ServerCache {
     // TODO: Change the below line to blockbook once available
```

Comment:
Should this line be removed now?

Reply:
i removed it.

Reply:
Looks good. Apply the fixup commit, and then merge.